styles/styles.css

```diff
@@ -0,0 +1,139 @@
+/* color1: #09B3C4 */
+/* color2: #e7b43f */
+/* color3: #d05664 */
+
+@import url('https://fonts.googleapis.com/css?family=Cabin|Homemade+Apple');
+
+/* general element rules */
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+/*general + body rules*/
+body {
+  font-family: 'Cabin', sans-serif;
+  font-size: 18px;
+  color: #09B3C4;
+}
+
+h1 a, h2, h4 {
+  font-family: 'Homemade Apple', cursive;
+}
+
+a {
+  color: #09B3C4;
+  text-decoration: none;
+}
+
+a:hover, a:visited:hover {
+  color: #e7b43f;
+}
+
+a:visited {
+  color: #d05664;
+}
+
+
+/*header rules*/
+header {
+  padding: 0 20%;
+  margin-bottom: 5%;
+}
+
+nav {
+  min-width: 1000px;
+}
+
+nav * {
+  display: inline;
+  text-align: center;
+}
+
+nav h1 {
+  padding: 0 2.5%;
+}
+
+.prof {
```

Comment:
does this refer to any of the classes in your html file? Can't find its reference anywhere